docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,336 @@
+
+ * WIP: Blueprint was approved and someone is working on it. Work in Progress.
+
+ * Implemented: This means the BP is already implemented and delivered to the
+   Avocado’s master branch.
+
+ * Rejected: Rejected status means the idea was not implemented because it
+   wasn't approved by everyone or has some techinical limitations.
+
+ * Deprecated: Deprecated means it was approved, implemented, and at some
+   point, makes no more sense to have it. For example, anything related to the
+   legacy runner. Usually, Deprecated means that it was replaced by something
+   else.
+
+As you can see, there is no status to accommodate any future change in a
+blueprint. Blueprints should not be “voided.” Any improvement on an old
+blueprint should be presented as a new blueprint, changing the status of the
+original to “deprecated”.
+
+Sections
+--------
+
+In order to facilitate the reading and understanding of the problem, all
+blueprints must have the following sections:
+
+ * TL;DR
+
+ * Motivation
+
+ * Specification
+
+ * Backwards Compatibility
+
+ * Security Implications
+
+ * How to Teach This
+
+ * Related Issues
+
+ * References
+
+Bellow a brief description of what you should write in each section:
+
+ * TL;DR: Should be a short description of your blueprint. Like an abstract. We
+   recommend writing this at the end of your first draft. This will give you a
+   better overview of it.
+
+ * Motivation: This should be the motivation of your proposed solution, not the
+   motivation of the blueprint itself. It describes the problem. Here, you
+   should answer "why" your solution is needed.
+
+ * Specification: In this section, you should describe how you are going to
+   solve the problem. You can create subsections here to organize your ideas
+   better. Please keep in mind that it is useful to mention the details, with
+   code snippets, examples, and/or references. This will save you time, making
+   sure that everyone is in agreement with the proposed solution.
+
+ * Backwards Compatibility: How is your proposal going to affect older versions
+   of Avocado? Should we deprecate some modules, classes, or methods? Are we
+   going to keep backwards compatibility or not?
+
+ * Security Implications: Do you have any concerns about security with your
+   proposed solution and what are they?
+
+ * How to Teach This: What is the best way to inform our devs and users about
+   your new feature/solution?  feature/solution?
+
+ * Related Issues: Here, you should mention Github links for both: a) current
+   open issues that are blocking while waiting for your BP and b) all open
+   issues that will render this BP as “implemented” when closed.
+
+   1. Issues to address this BP
+
+      Would be nice, if possible, to open issues on GH that covers all aspects
+      of your Blueprint.
+
+   2. Issues this BP will solve
+
+      What are the issues already existent on Avocado project that your
+      proposal will solve?
```

Comment:
Something that remains slightly unclear here: should the issues be just listed as #numbers or as links to GitHub?

Reply:
On the "_Related Issues_" description we have: "Here, **you should mention Github links for both**: a) current open issues that are blocking while waiting for your BP and b) all open issues that will render this BP as “implemented” when closed."